chaincode/src/services/PublicKeyService.ts

```diff
@@ -79,7 +79,8 @@ export class PublicKeyService {
 
     // check if we want the profile of the admin
     if (process.env.DEV_ADMIN_PUBLIC_KEY !== undefined && process.env.DEV_ADMIN_USER_ID !== undefined) {
-      const adminEthAddress = signatures.getEthAddress(process.env.DEV_ADMIN_PUBLIC_KEY);
+      const nonCompactPK = signatures.getNonCompactHexPublicKey(process.env.DEV_ADMIN_PUBLIC_KEY);
+      const adminEthAddress = signatures.getEthAddress(nonCompactPK);
```

Comment:
Would it make more sense here to try and cover both cases. Like having logic around seeing if it needs to be decompressed.

Reply:
Actually everything happens inside `getNonCompactHexPublicKey`. When it gets non-compact key, it returns the same value. Does it answer your question? Because I'm not sure if I fully understood

Reply:
Yup, thats exactly it. Thanks